compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/symbols/SymbolKind.java

```diff
@@ -44,5 +44,6 @@ public enum SymbolKind {
     ENUM_MEMBER,
     PARAMETER,
     PATH_PARAMETER,
-    ANNOTATION_ATTACHMENT
+    ANNOTATION_ATTACHMENT,
+    TUPLE_MEMBER_TYPE
```

Comment:
```suggestion
    TUPLE_MEMBER
```
?

Reply:
+1 and please check my thoughts on https://github.com/ushirask/ballerina-lang/pull/54#discussion_r1062132733